packages/p2p/src/components/my-ads/my-ads-row-renderer.jsx

```diff
@@ -259,7 +274,7 @@ const MyAdsRowRenderer = observer(({ row: advert, setAdvert }) => {
                                     className='p2p-my-ads__table-popovers__deactivate'
                                     message={localize('Deactivate')}
                                 >
-                                    <Icon icon='IcArchive' color={general_store.is_barred && 'disabled'} size={16} />
+                                    <Icon icon='IcArchive' color={general_store.is_barred && 'disabled'} />
                                 </Popover>
                             </div>
                         ) : (
```

Comment:
Similar to my previous comment, perhaps we can make it 1 div, and use the ternary on className, message and icon:

```
<div onClick={onClickActivateDeactivate}>
            <Popover
                alignment='bottom'
                className={`p2p-my-ads__table-popovers__${is_advert_active ? 'activate' : 'deactivate'}`}
                message={localize(`${is_advert_active ? 'Activate' : 'Deactivate'}`)}
            >
                <Icon icon={`${is_advert_active ? 'IcUnarchive': 'IcArchive'}`} 
                   color={general_store.is_barred && 'disabled'} 
                />
            </Popover>
</div>
```